packages/aragon-wrapper/src/index.js

```diff
@@ -151,7 +151,8 @@ export default class Aragon {
     const defaultOptions = {
       apm: {},
       defaultGasPriceFn: () => { },
-      provider: detectProvider()
+      provider: detectProvider(),
+      cacheDriver: null
```

Comment:
My main hesitation around this is what a proper API should look like, and I'm starting to think something like:

```js
cache: {
  forceLocalStorage: true
}
```

Might be enough. I assume nobody wants to use WebSQL (let's just remove it from the default list), and I don't know anything about its security implications (now that we know IndexedDB is a PITA), so let's just give users the ability to "force" the localstorage downgrade if they want to.